src/init.jl

```diff
@@ -64,7 +64,7 @@ function __init__()
             )
         end,
     )
-    if _default_backend() != :gr
+    if _default_backend() === :gr
```

Comment:
That wasn't a typo since, we need the requires only if GR is not the default backend

Reply:
Sorry, I got no notifications, because the PR was on draft.